src/links/node-modules-linker.js

```diff
@@ -103,7 +103,13 @@ export default class NodeModuleLinker {
   async _populateImportedComponentsLinks(component: Component): Promise<void> {
     const componentMap = component.componentMap;
     const componentId = component.id;
-    const bindingPrefix = this.consumer ? this.consumer.config.bindingPrefix : null;
+    if (!componentId.hasScope()) return; // when isolating new components// from isolated;
+    const bindingPrefix =
+      this.consumer && this.consumer.bitJson
+        ? this.consumer.bitJson.bindingPrefix
+        : this.consumer && this.consumer.config && this.consumer.config.bindingPrefix
+          ? this.consumer.config.bindingPrefix
+          : DEFAULT_BINDINGS_PREFIX;
```

Comment:
one of my changes done recently is renaming `consumer.bitJson` to `consumer.config`. So, no need to check for `this.consumer.bitJson` as it'll always be empty. I'd probably change this to:
```
const bindingPrefix = this.consumer ? this.consumer.config.bindingPrefix : DEFAULT_BINDINGS_PREFIX;
```